pop-api/examples/dao/src/tests.rs

```diff
@@ -0,0 +1,549 @@
+
+fn deploy_with_default(session: &mut Session<Pop>) -> Result<AccountId, Psp22Error> {
+	deploy(
+		session,
+		"new",
+		vec![TOKEN.to_string(), VOTING_PERIOD.to_string(), MIN_BALANCE.to_string()],
+	)
+}
+
+#[drink::test(sandbox = Pop)]
+fn new_constructor_works(mut session: Session) {
+	let _ = env_logger::try_init();
+	// Deploy a new contract.
+	let contract = deploy_with_default(&mut session).unwrap();
+	// Token exists after the deployment.
+	assert!(session.sandbox().asset_exists(&TOKEN));
+	// Successfully emit event.
+	assert_last_contract_event!(
+		&session,
+		Created {
+			id: TOKEN,
+			creator: account_id_from_slice(&contract),
+			admin: account_id_from_slice(&contract),
+		}
+	);
+}
+
+#[drink::test(sandbox = Pop)]
+fn join_dao_works(mut session: Session) {
+	let _ = env_logger::try_init();
+	let value = AMOUNT / 2;
+	// Deploy a new contract.
+	let contract = deploy_with_default(&mut session).unwrap();
+	session.set_actor(ALICE);
+	// Mint tokens and approve.
+	assert_ok!(session.sandbox().mint_into(&TOKEN, &ALICE, AMOUNT));
+	assert_ok!(session.sandbox().approve(&TOKEN, &ALICE, &contract.clone(), AMOUNT));
+	assert_eq!(session.sandbox().allowance(&TOKEN, &ALICE, &contract.clone()), AMOUNT);
+	assert_eq!(session.sandbox().balance_of(&TOKEN, &ALICE), AMOUNT);
+
+	// Alice joins the dao
+	assert_ok!(join(&mut session, value));
+
+	// Successfully emit event.
+	assert_last_contract_event!(
+		&session,
+		Transfer {
+			from: Some(account_id_from_slice(&ALICE)),
+			to: Some(account_id_from_slice(&contract)),
+			value,
+		}
+	);
+
+	// We check that Alice is a member with a voting power of 20000
+	if let Ok(member) = members(&mut session, ALICE) {
+		assert_eq!(member.voting_power, 20000);
+	}
+}
+
+#[drink::test(sandbox = Pop)]
+fn member_create_proposal_works(mut session: Session) {
+	let _ = env_logger::try_init();
+	// Deploy a new contract.
+	let contract = deploy_with_default(&mut session).unwrap();
+	// Prepare voters accounts
+	let _ = prepare_dao(&mut session, contract.clone());
+
+	// Alice create a proposal
+	let description = "Funds for creation of a Dao contract".to_string().as_bytes().to_vec();
+	let amount = AMOUNT * 3;
+	session.set_actor(ALICE);
+	let mut call_f = Vec::new();
+	let call0 = RuntimeCall::Fungibles(FungiblesCall::TransferFrom {
+		token: TOKEN,
+		from: account_id_from_slice(&contract),
+		to: account_id_from_slice(&BOB),
+		value: amount,
+	})
+	.encode(&mut call_f);
+	assert_ok!(create_proposal(&mut session, BOB, amount, description, call_f));
+
+	assert_last_contract_event!(
+		&session,
+		Created {
+			id: 1,
+			creator: account_id_from_slice(&ALICE),
+			admin: account_id_from_slice(&contract),
+		}
+	);
+
+	// Another proposal created by Bob
+	let description_bis =
+		"Funds for creation of another Dao contract".to_string().as_bytes().to_vec();
+	session.set_actor(BOB);
+	let mut call_f = Vec::new();
+	let call0 = RuntimeCall::Fungibles(FungiblesCall::TransferFrom {
```

Comment:
```suggestion
	RuntimeCall::Fungibles(FungiblesCall::TransferFrom {
```

Seems like the variable is not used anywhere